backend/tests/data_layer/database/crud/postgresql/test_user_crud.py

```
@@ -0,0 +1,242 @@
+    Verifies that `is_attr_data_in_db` detects existing attributes.
+    """
+    # Test: 3.1 ( Verify the email already exists )
+    result = is_attr_data_in_db(User, {"email": "testuser@gmail.com"}, session=session)
+    assert result == "email already exists"
+
+    # Test: 3.2 ( Verify the email does not exist )
+    assert (
+        is_attr_data_in_db(User, {"email": "sampleuser@example.com"}, session=session)
+        is None
+    )
+    with pytest.raises(HTTPException) as exc:
```

Ruff (0.8.2)

Local variable `exc` is assigned to but never used

Remove assignment to unused variable `exc`

(F841)